### ISSUE TITLE
ToolbarSeparator: Accessibility issue aria-label attribute cannot be used on a div with no valid role attribute.

### ISSUE THREAD

**Issue by achuth2912** [SAP/ui5-webcomponents-react#5808]
### Describe the bug

When axe tool is run on the page,  Axe tool detects error 'Elements must only use permitted ARIA attributes'
aria-label attribute cannot be used on a div with no valid role attribute.

Attaching the Screenshot for reference.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/layouts-floorplans-toolbar--with-separator

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Elements must only use allowed ARIA attributes

### Screenshots or Videos

<img width="1728" alt="Screenshot 2024-05-15 at 6 00 50 PM" src="https://github.com/SAP/ui5-webcomponents-react/assets/71249041/58a59654-42b6-40f0-b9c4-cefa1f631705">

### UI5 Web Components for React Version

1.23.0

### UI5 Web Components Version

1.20.0

### Browser

Chrome

### Operating System

Mac OS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
